packages/suite/src/views/wallet/send/components/Options/components/BitcoinOptions/index.tsx

```diff
@@ -137,6 +149,31 @@ const BitcoinOptions = () => {
                             <OnOffSwitcher isOn={broadcastEnabled} />
                         </StyledButton>
                     </Tooltip>
+                    {!utxoSelectionEnabled && (
+                        <Tooltip
+                            guideAnchor={instance => (
+                                <OpenGuideFromTooltip
+                                    id="/suite-basics/send/coin-selection.md"
+                                    instance={instance}
+                                />
+                            )}
+                            content={<span>Coin selection tooltip...</span>}
+                            cursor="pointer"
+                        >
+                            <StyledButton
+                                variant="tertiary"
+                                icon="COIN_CONTROL"
+                                onClick={() => {
+                                    // open additional form
+                                    toggleOption('utxoSelection');
+                                    composeTransaction();
+                                }}
+                            >
+                                <Translation id="TR_COIN_CONTROL" />
+                                {isCoinControl && <OnOffSwitcher isOn />}
```

Comment:
```suggestion
                                <OnOffSwitcher isOn={isCoinControl} />
```

Not sure but would be ok for consistency

Reply:
I would leave it as it is, otherwise it looks bit weird in default state (OFF), red colour indicate some state worthy of attention
<img width="641" alt="image" src="https://user-images.githubusercontent.com/3729633/188173083-0bdc9acf-554f-4fac-9756-4530944edab0.png">